src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -443,5 +444,19 @@ public static LSP.ReferenceKind[] SymbolUsageInfoToReferenceKinds(SymbolUsageInf
 
             return referenceKinds.ToArrayAndFree();
         }
+
+        public static string ProjectIdToProjectContextId(ProjectId id)
+        {
+            return id.Id + "|" + id.DebugName;
```

Comment:
should we be using a debug name as part of the id?  if so, should we rename it, as it now matters for actual scenarios?

Reply:
For comparison it's still debug only -- this was just so when you end up with a ProjectId on the other side it has the same debug name as you started with.